Dockerfile.pr

```diff
@@ -7,6 +7,13 @@ ENV NEXT_OUTPUT_STANDALONE=true
 ENV NEXT_PUBLIC_ADDRESSCOMPLETE_API_KEY=UR78-BU29-RU35-EP49 
 # Address Complete API key is domain locked, and provided to the client browser, where the API is processed.
 
+# 6 MB
+ENV NEXT_PUBLIC_BODY_SIZE_LIMIT_WITH_FILES=6291456
```

Comment:
@bryan-robitaille 

If you can review what's still needed here

Reply:
removed max file size and added other env args to docker build arguments for all app environments